### PR TITLE
Fix/care charges assessed

### PR DESCRIPTION
**Pull request by mugnivenko**
https://app.clickup.com/t/1vrgpkb

cases
S117: care-package/5b7a5e88-c4bb-44c8-aaf3-68de0a361e4e/broker-care-charges
hasAssessment: care-package/c91a1633-7503-4816-a91f-ccf3a585fdc2/broker-care-charges
update care charge: care-package/624170f8-06c4-4bde-be5c-b73591a229d9/broker-care-charges

### CODE REVIEW

File: components/Pages/CarePackages/BrokerCareCharge/PreviousCareCharge.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Announcement, Button, Container, HorizontalSeparator, VerticalSeparator } from '../../../HackneyDS';
+
+export const PreviousCareCharges = ({ usePreviousCareCharge, useNewCareCharge }) => (
+  <Announcement
+    className='warning'
```
Comment (akeamwoodsND) [inline components/Pages/CarePackages/BrokerCareCharge/PreviousCareCharge.js:6]: use the isError prop as this was designed

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
Changes i've raised in the first one haven't be responded or fixed. The code you have here is much harder to read  than what was originally wrote. It is better to have two simple solutions than one over-engineered solution.

File: components/Pages/CarePackages/BrokerCareCharge/PreviousCareCharge.js (hunk 1 above)
Comment (mugnivenko) [reply components/Pages/CarePackages/BrokerCareCharge/PreviousCareCharge.js:6]: this is not error announcement
https://www.figma.com/file/Jc6Pxo05rghhH47fcDt9mm/Finance?node-id=7899%3A104010

File: components/HackneyDS/Announcement/index.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ const mainClass = 'lbh-announcement';
 export default function Announcement({ title, children, className, isError }) {
   const isErrorClass = isError ? ` ${mainClass}--error` : '';
   return (
-    <section className={`${mainClass} ${mainClass}--site${className && ` ${className}`}${isErrorClass}`}>
+    <section className={`${mainClass} ${mainClass}--site${className ? ` ${className}` : ''}${isErrorClass}`}>
```
[the remark this answers is not in the document]
Comment (mugnivenko) [reply components/HackneyDS/Announcement/index.js:8]: fixed

File: components/Pages/CarePackages/BrokerCareCharge/PreviousCareCharge.js (hunk 1 above)
Comment (akeamwoodsND) [reply components/Pages/CarePackages/BrokerCareCharge/PreviousCareCharge.js:6]: So then remove the class altogether?

Comment (akeamwoodsND) [reply components/Pages/CarePackages/BrokerCareCharge/PreviousCareCharge.js:6]: And it as a prop

**Comment by mugnivenko** [comment]
fixed:
1) buttons
2) WarningText component
3) refactoring Announcement component

File: components/Pages/CarePackages/BrokerCareCharge/PreviousCareCharge.js (hunk 1 above)
Comment (mugnivenko) [reply components/Pages/CarePackages/BrokerCareCharge/PreviousCareCharge.js]: done

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
cannot click on anything on care charge page because of overflow
Screenshot 2021-12-06 at 10 21 01

**Comment by mugnivenko** [comment]
> cannot click on anything on care charge page because of overflow <img alt="Screenshot 2021-12-06 at 10 21 01" width="1440" src="https://user-images.githubusercontent.com/89010589/144829389-a17b2e90-9c19-4b93-bec4-bfc3c4eb81ae.png"

**Pull request closed without merging**